open-sphere-plugins/infinity/src/main/java/io/opensphere/infinity/json/Bucket.java

```diff
@@ -1,10 +1,10 @@
 package io.opensphere.infinity.json;
 
 /** Elasticsearch bucket JSON bean. */
-public class Bucket
+public class Bucket<T>

```

Comment:
Document the type parameter in the class javadoc (e.g.: @param <T> something something